src/Build/BackEnd/Components/Communications/NodeProviderOutOfProcBase.cs

```diff
@@ -466,8 +465,8 @@ private Process LaunchNode(string msbuildLocation, string commandLineArgs)
                 creationFlags |= BackendNativeMethods.CREATE_NEW_CONSOLE;
             }
 
-            BackendNativeMethods.SECURITY_ATTRIBUTES processSecurityAttributes = new BackendNativeMethods.SECURITY_ATTRIBUTES();
-            BackendNativeMethods.SECURITY_ATTRIBUTES threadSecurityAttributes = new BackendNativeMethods.SECURITY_ATTRIBUTES();
+            BackendNativeMethods.SECURITY_ATTRIBUTES processSecurityAttributes = new();
```

Comment:
These could be moved into the `else` block below where its actually used, unless we get rid of the different ways of launching the process in which case this can just go away.